update-center-source.properties

```diff
@@ -1,4 +1,4 @@
-plugins=abap,aemrules,android,ansible,sonarapex,apigee,authaad,authbitbucket,authgithub,authgitlab,authgoogleoauth,authoidc,authsaml,branch,checkstyle,citymodel,clover,cobertura,cobol,codecrackercsharp,communitybsl,cpp,crowd,csharp,cssfamily,developer,findbugs,flex,fxcop,github,gitlab,go,governance,groovy,guavamigration,ha,icode,issueresolver,jacoco,java,javai18n,javascript,jdepend,kotlin,ldap,license,lits,l10nde,l10nfr,l10nel,l10nes,l10nit,l10nja,l10nko,l10npt,l10nru,l10nzh,lua,mutationanalysis,pam,php,pitest,pli,plsql,pmd,python,quboo,shellcheck,status,rci,rpg,ruby,sonarscala,scmcvs,scmgit,scmjazzrtc,scmtfvc,scmmercurial,scmperforce,scmsvn,scmclearcase,smells,softvis3d,Sonargraph,sonargraphintegration,swift,timeline,tsql,typescript,webdriver,vb,vbnet,web,xanitizer,xml,yaml
+plugins=abap,aemrules,android,ansible,sonarapex,apigee,authaad,authbitbucket,authgithub,authgitlab,authgoogleoauth,authoidc,authsaml,branch,checkstyle,citymodel,clover,cobertura,cobol,codecrackercsharp,communitybsl,cpp,crowd,csharp,cssfamily,developer,findbugs,flex,fxcop,github,gitlab,go,governance,groovy,guavamigration,ha,icode,issueresolver,jacoco,java,javai18n,javascript,jdepend,kotlin,ldap,license,lits,l10nde,l10nfr,l10nel,l10nes,l10nit,l10nja,l10nko,l10npt,l10nru,l10nzh,lua,mutationanalysis,pam,php,pitest,pli,plsql,pmd,python,quboo,shellcheck,status,rci,rpg,ruby,sonarscala,scmcvs,scmgit,scmjazzrtc,scmtfvc,scmmercurial,scmperforce,scmsvn,scmclearcase,smells,softvis3d,Sonargraph,sonargraphintegration,swift,timeline,tsql,typescript,webdriver,vb,vbnet,web,xanitizer,xml,yaml,mybatis
```

Comment:
Please keep this list alphabetical rather than inserting `mybatis` at the end

Reply:
I have fixed it.